pkg/cmd/pr/shared/survey_test.go

```diff
@@ -124,6 +128,63 @@ func TestMetadataSurvey_keepExisting(t *testing.T) {
 	assert.Equal(t, []string{"The road to 1.0"}, state.Projects)
 }
 
+// TODO projectsV1Deprecation
+// Remove this test and projectsV1MetadataFetcherSpy
+func TestMetadataSurveyProjectV1Deprecation(t *testing.T) {
+	t.Run("when projectsV1 is supported, requests projectsV1", func(t *testing.T) {
+		ios, _, _, _ := iostreams.Test()
+		repo := ghrepo.New("OWNER", "REPO")
+
+		fetcher := &projectsV1MetadataFetcherSpy{}
+		pm := prompter.NewMockPrompter(t)
+		pm.RegisterMultiSelect("What would you like to add?", []string{}, []string{"Assignees", "Labels", "Projects", "Milestone"}, func(_ string, _, options []string) ([]int, error) {
+			i, err := prompter.IndexFor(options, "Projects")
+			require.NoError(t, err)
+			return []int{i}, nil
+		})
+		pm.RegisterMultiSelect("Projects", []string{}, []string{"Huge Refactoring"}, func(_ string, _, _ []string) ([]int, error) {
+			return []int{0}, nil
+		})
+
+		err := MetadataSurvey(pm, ios, repo, fetcher, &IssueMetadataState{}, gh.ProjectsV1Supported)
+		require.ErrorContains(t, err, "expected test error")
+
+		require.True(t, fetcher.projectsV1Requested, "expected projectsV1 to be requested")
+	})
+
+	t.Run("when projectsV1 is supported, does not request projectsV1", func(t *testing.T) {
```

Comment:
Should be "...is *not* supported...".

Reply:
Yes, will address after stack is merged.